source/geod24/LocalRest.d

```diff
@@ -282,6 +282,35 @@ public void sleep (Duration timeout)
     cond.wait(timeout);
 }
 
+/*******************************************************************************
+
+    Run an asynchronous task after the set time in LocalRest's
+    can set it to repeat or run it only once.
+    Works similar to the setTimer of vibe.d.
+
+    Params:
+        timeout = Determines the minimum amount of time that elapses before
+            the timer fires.
+        dg = If non-null, this delegate will be called when the timer fires
+        periodic = Speficies if the timer fires repeatedly or only once
+
+*******************************************************************************/
+
+public void setTimer (Duration timeout, void delegate() dg,
+    bool periodic = true)
```

Comment:
[setTimer](https://vibed.org/api/vibe.core.core/setTimer) is non-periodic by default, and I think that makes more sense.

Reply:
It's with periodic focus, but I'll apply the suggestion.